<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatMigration</class>
 <widget class="QDialog" name="MatMigration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bat Migration</string>
  </property>
  <widget class="QPlainTextEdit" name="accessToken">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Access Token</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Document of files to migrate</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="filesDocument">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="stationBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Station</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="yearBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QPushButton" name="migrateButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Migrate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileSelectButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
  </widget>
  <widget class="QLabel" name="curr_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
